### PR TITLE
feat: added Snackbar components

### PR DESCRIPTION
**Pull request by kjais1720**
Added 2 Snackbar components
- A standard snackbar
- A stacked snackbar
- Preview link👇
https://deploy-preview-18--tarang-ui.netlify.app/docs/snackbars/snackbar

### CODE REVIEW

**Review by ttahm3d — APPROVED** [review]
Fix the typo and merge

**Review by vivekmehto — APPROVED** [review]
Looks Perfect.

**Review by shubhamkumar648 — APPROVED** [review]
code looks good to me ,  suggestion:-you can also used flex flow proerty

File: Docs/Snackbars/snackbar.html [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+                        <button class="tr-snackbar-link">Undo</button>
+                        <button class="close-icon"><i class="fas fa-times-circle"></i></button>
+                    </div>
+                </div>
+
+                <div id="preview-block-2" class="hide preview-block">
+                    <div class="tr-snackbar d-flex f-wrap tr-snackbar-stacked">
+                        <p class="tr-snackbar-text">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quos, consequuntur! Delectus</p>
+                        <button class="tr-snackbar-link">Action-1</button>
+                        <button class="tr-snackbar-link">Acion-2</button>
```
[the remark this answers is not in the document]
Comment (kjais1720) [reply Docs/Snackbars/snackbar.html]: Thank you for pointing it out, I have fixed it in the latest commit

**Pull request merged**